src/namespaces.json

```
@@ -231,5 +231,17 @@
     "min": 0,
     "invalid": [],
     "visible": false
+  },
+  "lao": {
+    "key": "lao",
+    "symbol": "LAO",
+    "name": "LAO",
```

I'd suggest to put the full name here
```suggestion
    "name": "The LAO",
```